modules/gdscript/gdscript.cpp

```diff
@@ -930,17 +936,23 @@ bool GDScript::_set(const StringName &p_name, const Variant &p_value) {
 				Variant value = p_value;
 				if (member->data_type.has_type && !member->data_type.is_type(value)) {
 					const Variant *args = &p_value;
-					Callable::CallError err;
-					Variant::construct(member->data_type.builtin_type, value, &args, 1, err);
-					if (err.error != Callable::CallError::CALL_OK || !member->data_type.is_type(value)) {
+					Callable::CallError ce;
+					Variant::construct(member->data_type.builtin_type, value, &args, 1, ce);
+					if (ce.error != Callable::CallError::CALL_OK || !member->data_type.is_type(value)) {
```

Comment:
I just wanted to be consistent, but there are also `err` and `call_err` variables in this file. Not sure which is better.

Reply:
Fixed it :)

Reply:
You probably meant `has_valid_getter` copy-paste mistake.

Reply:
Wait, I'm confused now. Is line 1337 (lol) supposed to set has_valid_setter or has_valid_getter? I figured it should be has_valid_setter because the nearby code is about setters.

Reply:
This comment is **not** on line 1337 (it's above, marked as "Outdated" and you've corrected it). This comment is about naming `CallError` variables (`err` vs `ce` vs `call_err`).